packages/database/src/core/RepoInfo.ts

```diff
@@ -15,13 +15,17 @@
  * limitations under the License.
  */
 
-import { assert } from '@firebase/util';
+import { assert, EmulatorMockTokenOptions } from '@firebase/util';
 
 import { LONG_POLLING, WEBSOCKET } from '../realtime/Constants';
 
 import { PersistentStorage } from './storage/storage';
 import { each } from './util/util';
 
+export interface RepoInfoEmulatorOptions {
```

Comment:
Can we re-use this type in `api/Database.ts`, line 350?

Reply:
That would require an public API change, unfortunately.